packages/remirror__extension-link/package.json

```diff
@@ -42,10 +42,12 @@
     "@babel/runtime": "^7.13.10",
     "@remirror/core": "^1.4.3",
     "@remirror/extension-events": "^1.1.1",
-    "@remirror/messages": "^1.0.6"
+    "@remirror/messages": "^1.0.6",
+    "extract-domain": "2.2.1"
```

Comment:
why do we want `extract-domain@2.2.1` instead of the latest `^2.4.1`?

Reply:
Using the TLD option, in these version is [considerably slower](https://github.com/bjarneo/extract-domain#tld-support)